hack/update-proto.sh

```diff
@@ -18,6 +18,9 @@ HACK_DIR=$(dirname "${BASH_SOURCE}")
 
 echo "Updating plugin proto"
 
+protoc pkg/plugin/proto/backupitemaction/v1/*.proto --go_out=plugins=grpc:pkg/plugin/generated/ -I pkg/plugin/proto/
 protoc pkg/plugin/proto/*.proto --go_out=plugins=grpc:pkg/plugin/generated/ -I pkg/plugin/proto/
+cp -rf pkg/plugin/generated/github.com/vmware-tanzu/velero/pkg/plugin/generated/* pkg/plugin/generated
+rm -rf pkg/plugin/generated/github.com
```

Comment:
Could you add more context for the `cp` and `rm`? I remember we talked about them in the slack channel or issue, it's better to link them here to help understand the backgroup

Reply:
@ywk253100 The issue is that the way it's set up currently, the generated code gets put in the wrong place with those duplicated directories. The workaround for now is to move them into the right place. Ideally we'll figure out how to tell the code generator where to put them. For some reason these are showing up in the wrong place, while same-package files are fine. I think we have a github issue for that which I'm assigned to. I hope to start investigating that soon.